.scalafmt.conf

```diff
@@ -0,0 +1,53 @@
+version = 3.8.3
+runner.dialect = scala3
+maxColumn = 120
+lineEndings = unix
+importSelectors = binPack
+
+
+project {
+ git = true
+}
+
+
+align {
+ preset = most
+ tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"},
+ "=", "<-", "->", "%", "%%", "should", "shouldBe", "shouldEqual", "shouldNot", "must" ]
+ arrowEnumeratorGenerator = true
+ openParenCallSite = false
+ openParenDefnSite = false
+}
+
+
+binPack {
+ parentConstructors = false
+}
+
```

Comment:
Do we need two empty spaces between each section on this page?

Reply:
Removed.